### PR TITLE
Module2

### PR DESCRIPTION
**Pull request by SpaceForCreativity**
Bad pull request with many gaps because of not enough time. I'll fill all gaps at the weekends

### CODE REVIEW

File: src/auth/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import './Login.css';
```
Comment (NikitaSedyx) [inline src/auth/Login/Login.jsx:1]: it's up to you, but usually third party is imported, than some aliases/common/etc and after that module specific

and would be nice to use css modules

File: src/auth/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  }
+
+  return (
+    <div className="login-container">
+      <h1>Login</h1>
+      <Input
+        labelText="Email"
+        placeholder="Enter email"
+        onChange={(e) => setMail(e.target.value)}
```
Comment (NikitaSedyx) [inline src/auth/Login/Login.jsx:25]: you again mix controlled and uncontrolled approaches

File: src/common/hooks/utils/useLocalStorage.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+const getLocalValue = (key, initValue) => {
+  const localValue = JSON.parse(localStorage.getItem(key));
+
+  if (localValue) return localValue;
+  if (initValue instanceof Function) return initValue();
+  return initValue;
+};
+
+export default function useLocalStorage(key, initValue) {
```
Comment (NikitaSedyx) [inline src/common/hooks/utils/useLocalStorage.js:11]: https://github.com/rehooks/local-storage

File: src/auth/Login/Login.jsx (hunk 1 above)
Comment (SpaceForCreativity) [reply src/auth/Login/Login.jsx:1]: What do you mean?

Comment (NikitaSedyx) [reply src/auth/Login/Login.jsx:1]: order of imports, third part, aliases/common/etc and after that module specific

Comment (SpaceForCreativity) [reply src/auth/Login/Login.jsx:1]: got it, thanks

**Pull request merged**